Sources/BraintreeCore/Analytics/AnalyticsSendable.swift

```diff
@@ -0,0 +1,7 @@
+/// Describes a class that batches and sends analytics events.
+/// - Note: Specifically created to mock the `BTAnalyticsService` singleton.
+protocol AnalyticsSendable: AnyObject {
```

Comment:
🫶🏻

Reply:
If you have a better name suggestion for this, please let me know